packages/calcite-components/src/components/input/input.scss

```diff
@@ -18,10 +18,18 @@
  * @prop --calcite-input-placeholder-text-color: defines the color of placeholder text in the component.
  * @prop --calcite-input-prefix-background-color: defines the background color of the prefix sub-component.
  * @prop --calcite-input-prefix-text-color:  defines the text color of the prefix sub-component.
+ * @prop --calcite-input-progress-background-color: Defines the background color of the progress sub-component.
+ * @prop --calcite-input-progress-fill-color: Defines the background color of the progress sub-component bar.
```

Comment:
@alisonailea @jcfranco are we using "fill-color" or should this be more like `--calcite-input-progress-bar-background-color`? I don't think we need to distinguish "fill" from "background-color" as its more of an implementation detail. IMO this is really just the background color of the "bar".

Reply:
I do see that we use `-fill-color` in `meter` and `rating` components as well. Just wanted to bring this up.

Reply:
On the other hand, since bar is the active color and background is the grey stretch, it's sort of intuitive to call it fill, visually, from user perspective. I think if the point of name change is to hide the implementation details, fill does it better.

Reply:
I think `fill` works and is already being used. It looks like it's common too: https://spectrum.adobe.com/page/progress-bar/#Anatomy.

I think it'll be worth it to do an audit once the epic PR is close to ensure all props are consistently named. cc @alisonailea

Reply:
sounds good

Reply:
I would agree fill is a more accurate name. Although I wonder if `--calcite-progress-bar-color` and `--calcite-progress-color` wouldn't be even more accurate?

Reply:
It just seems like `fill-color` and `background-color` are pretty vague. Like what is being filled? How does it differ from a `background-color` of a specific thing inside the component.